### ISSUE TITLE
Admin Master Pairings List

### ISSUE THREAD

**Issue by rochellesunshine** [arcanis/secretsanta#8]
Is there a way to export a master pairings csv file to keep a record of all the secret santa assignments?

**Comment by swishi** [comment]
This would realy be a great option to improve the script! I'm looking forward!

Take care 
Ismael

**Comment by elin1923** [comment]
I agree! this would be an amazing feature for a large group where not everyone has a digital form of contact or the group is too large and emailing the pairings is too difficult.

**Comment by arcanis** [comment]
You can now download a CSV of the generated links
